runner/runner.go

```diff
@@ -155,15 +155,18 @@ func getSchedule(tests []string) ([]string, error) {
 		shardID     = os.Getenv("CIRCLE_NODE_INDEX")
 		requestBody = &bytes.Buffer{}
 	)
+	fmt.Printf("getSchedule: %v", tests)
```

Comment:
Nope. Removed, but kept the one that provides more information if test args fail to parse.